### ISSUE TITLE
Docs: Missing all the environment variable options for registry Docker image

### ISSUE THREAD

**Issue by melroy89** [distribution/distribution#4298]
### Description

Could somebody please [document](https://distribution.github.io/distribution) the environment variable options that are ALL available when using the Docker image registry.

I just noticed: https://github.com/distribution/distribution/issues/4151#issuecomment-1802394030 .. Maybe you want to explicit document this somewhere instead of a comment.

**Comment by milosgajdos** [comment]
Feel free to open a PR.

**Comment by melroy89** [comment]
That is a bit of an issue. I don't know all the environment variables. A bit of a chicken egg situation.

**Comment by melroy89** [comment]
Yes I know about that. But this issue is again about listing all the available options / environment variables.

**Comment by milosgajdos** [comment]
That wouldnt scale I'm afraid.
